### DISCUSSION TITLE
How do I dupe the Dwindle wiki page?

### DISCUSSION THREAD

**Discussion by bearcatsandor** [hyprwm/Hyprland#344] [Q&A]
I love Hyprland! As exciting as the animations displayed on the Dwindle page in the wiki, adding a Dwindle section and turning it on with psuedotile=1 does nothing.

Would someone mind posting the part of a config that will enable the behavior shown on the Dwindle page in the wiki?

Thank you.

**Comment by vaxerski** [comment]
the dwindle section cannot be "enabled"...?

It's just a set of settings controlling the behavior of the dwindle layout. 

No clue what you are talking about.

**Reply by bearcatsandor** [reply]
"Dwindle Layout" is it's own section under configuration, which makes it seem as though it's an alternate layout that one would enable. How does one do this?

**Reply by vaxerski** [reply]
its the default and only one

**Reply by bearcatsandor** [reply]
Ah, thank you. Do you know of a dotfile that duplicates the window behavior and look as seen on that page?

**Reply by vaxerski** [reply]
in the gifs? its default

**Comment by bearcatsandor** [comment]
I've replaced my config with the one shipped in the git repo, and still when I press SUPER + left mouse button and nothing happens. I'm expecting to move windows with my mouse. What am I missing?

Thank you

my config: https://bitbucket.org/bearcatsandor/workspace/snippets/BgzaGr

**Reply by fufexan** [reply]
Windows won't move, the focus will change to another window. This is only visible if you have more than 1 window open, or if you have 2 monitors and at least 1 window open on each.

**Reply by vaxerski** [reply]
> press SUPER + left mouse button

you need to hold it.

**Comment by bearcatsandor** [comment]
From the wiki page "Dwindle allows for mouse moving and resizing using the main_mod and the mouse (left for move, right for resize)  oving (main_mod + LMB)",  Under that there is a gif of moving the windows via mouse in the grid. Right?

**Reply by fufexan** [reply]
Correct.

**Reply by bearcatsandor** [reply]
Alright, well.  Just in case Gentoo's wlroots intsatallation was interfering wth your builtin one, I uninstalled gentoo's verion and there was no diference. There's sometning amiss, because I can't start gimp or rofi, and I can use both under Sway, and i can't move or re-size windows with my mouse.  Any ideas where to look?
